### PR TITLE
[ERSSUP-72623]-[DW]-[Add new content to A008 regarding application-restricted access]-[AST]

### PR DESCRIPTION
**Pull request by adamtallentire-nhsd**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/components/stu3/schemas/endpoints/a008-retrieve-worklist.yaml [hunk 1]
```diff
@@ -152,4 +172,4 @@ responses:
   '429':
     $ref: '../responses/TooManyRequests.yaml'
   '500':
-    $ref: '../responses/InternalServerError.yaml'
```
Comment (nhsd-david-wass) [inline specification/components/stu3/schemas/endpoints/a008-retrieve-worklist.yaml:155]: you may get a warning/error about file not ending with a blank line

**Review by nhsd-david-wass — DISMISSED** [review]
LGTM apart form 1 comment

**Review by adamoldfield-nhs — APPROVED** [review]
I wasn't able to view in Bloomreach due to HSCN VPN issues. But review all looks good.

**Comment by nhsd-david-wass** [comment]
merged on feature/ERSSUP-73552-74395-73911

**Pull request closed without merging**